### DISCUSSION TITLE
How to automatically minimize the terminal that accompanies startup?

### DISCUSSION THREAD

**Discussion by yk0n9** [tauri-apps/tauri#6911] [Q&A] [answered]
When the Tauri program is started, it will be accompanied by a terminal. How to make the terminal automatically minimize when it starts?

**Comment by FabianLars** [comment]
There isn't supposed to be a terminal 🤔 Tauri's default template include `#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]` at the top of the main.rs file to prevent this, and Tauri's `Command` api sets the `CREATE_NO_WINDOW` flag so that commands don't spawn any windows either.

Is there something you're doing that isn't covered by this?

**Reply by yk0n9** [reply, marked as answer]
Oh, I've solved it, instead of disabling the terminal, it auto-minimizes. I call this system function when registering HWND for Tauri. This is what I need.
```rust
unsafe {
        let hwnd = GetConsoleWindow();
        ShowWindow(hwnd, SW_HIDE);
}
```
